PhilanthropyDataCommons/pdc
# Set up TypeScript

This PR sets up TypeScript along with all of the other development environment tooling.

The README goes over the new commands available in the project.

To test this try the following:

```
npm ci
npm run lint
npm run test
npm run build
```

If you want to get fancy you can try writing some invalid typescript in `src/index.ts` and see what happens when you do those steps.
